crates/ra_project_model/src/lib.rs

```diff
@@ -191,6 +190,8 @@ impl ProjectWorkspace {
                             crate_graph.add_crate_root(
                                 file_id,
                                 edition,
+                                // FIXME json definitions can store the crate name
```

Comment:
Yeah, I am working on vscode config for `OUT_DIR` and just find out there is no crate name in json project definitions. :)